.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "vagrant-cookbook"]
```

Comment:
I guess this line should also be renamed, if it can.

Reply:
Not sure if that's possible. I did a `git mv vagrant-cookbook cookbook` and it just updated by itself.

Reply:
I tried renaming it, let's see what the CI thinks :)